diff -urN attr-1.0.1.org/configure.in attr-1.0.1/configure.in
--- attr-1.0.1.org/configure.in	Tue May  1 22:33:54 2001
+++ attr-1.0.1/configure.in	Tue May  1 22:39:07 2001
@@ -108,13 +111,6 @@
 dnl also check if man page source is gzipped
 dnl (usually on Debian, but not Redhat pre-7.0)
 have_zipped_manpages=false
-for d in ${prefix}/share/man ${prefix}/man ; do
-    if test -f $d/man1/man.1.gz
-    then
-	have_zipped_manpages=true
-	break
-    fi
-done
 AC_SUBST(have_zipped_manpages)
 
 dnl build definitions for use in Makefiles
